README.md

```diff
@@ -1,2 +1,13 @@
 # apis
 Example APIs designed to support the MaaS ecosystem.
+
+## Running the server
+To run this server, use the following steps:
+
+1. clone this repository
+2. set up a virtual environment
+3. install the `requirements.txt`
```

Comment:
Not too familiar with Python package managers. Is there a terminal command for this?

Reply:
Added a link to Python 3 documentation.